MAUI/Barcode-Generator/one-dimensional-symbology.md

```diff
@@ -0,0 +1,450 @@
+---
+layout: post
+title: One-dimensional symbology in .NET MAUI Barcodes (SfBarcodeGenerator) | Syncfusion
+description: Learn here all about one dimensional symbology of Syncfusion .NET MAUI Barcodes(SfBarcodeGenerator) control with key features and more.
+platform: maui
+control: SfBarcodeGenerator
+documentation: ug
+---
+
+# One-dimensional symbology in .NET MAUI Barcodes (SfBarcodeGenerator)
+
+The one-dimensional barcode represents the data by varying the widths and spacings of the parallel lines. These barcode types are also known as linear barcode types. The Syncfusion .NET MAUI barcode generator supports the following one-dimensional barcode types:
+
+* `Codabar`
+* `Code39`
+* `Code39Extended`
+* `Code93`
+* `UPCA`
+* `UPCE`
+* `EAN8`
+* `EAN13`
+* `Code128`
+* `Code128A`
+* `Code128B`
+* `Code128C`
+
+## Codabar
+
+* `Codabar` is a discrete numeric symbology that is used in libraries, blood banks and a variety of other information processing applications.
+* Encodes only numeric characters and some special characters like dash (-), colon (:), slash (/), plus (+).
```

Comment:
Encodes only numeric characters and some special characters such as dash (-), colon (:), slash (/), and plus (+).

Reply:
changes updated